projects/packages/changelogger/src/AddCommand.php

```diff
@@ -287,6 +287,9 @@ function ( $v ) {
 			} else {
 				if ( null === $entry ) {
 					$output->writeln( '<error>Entry must be specified in non-interactive mode.</>' );
+					if ( null === $entry && 'patch' === $significance ) {
```

Comment:
We already checked `null === $entry` on line 288 which wraps this, no need to repeat it here.
```suggestion
					if ( 'patch' === $significance ) {
```